packages/stress/esbuild.config.mjs

```diff
@@ -0,0 +1,22 @@
+import { build } from "esbuild";
+import { wasmLoader } from "esbuild-plugin-wasm";
+
+build({
+  entryPoints: ["./src/start.ts", "./src/demo.ts"],
+  bundle: true,
+  sourcemap: "inline",
+  platform: "node",
+  target: "node20",
+  format: "cjs",
+  outdir: "dist",
+  outExtension: { ".js": ".cjs" },
+  plugins: [wasmLoader()],
```

Comment:
An alternative to address the olm.wasm being renamed is to pass { mode: "embedded" } to the wasmLoader and the WASM will be inlined into the output js.

Reply:
We tried that and it didn’t work.